react/features/app-settings/functions.native.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+
+import { Dimensions, Platform } from 'react-native';
+
+const IPHONE_OFFSET = 20;
+const IPHONEX_HEIGHT = 812;
+const IPHONEX_OFFSET = 44;
+const IPHONEX_WIDTH = 375;
+
+/**
+* FIXME: Settings screen uses a custom implementation of safe views
+* to avoid empty areas on the screen when there should be a header rendered.
+* In long term we need to use a generic solution to handle "safe view
+* areas" for various devices. It can be React 0.50's {@code SafeAreaView} or
+* a module like 'react-native-safe-area-view' or a custom implemented solution.
+*/
+
+/**
+* Determines if the device is an iPad or not.
+*
+* @returns {boolean}
+*/
+export function isIPad() {
+    const {
+        height: screenHeight,
+        width: screenWidth
+    } = Dimensions.get('window');
+
+    return Platform.OS === 'ios' && (
+        Math.max(screenHeight, screenWidth)
+            / Math.min(screenHeight, screenWidth)) < 1.6;
+}
+
+/**
+* Determines if it's an iPhone X or not to handle notch on the settings
+* screen properly.
+*
+* @returns {boolean}
+*/
+function _isIPhoneX() {
+    const {
+        height: screenHeight,
```

Comment:
I don't think the rename adds anything here, it's used right below and the context is clear enough.